src/components/checkbox/checkbox.ts

```diff
@@ -192,6 +199,15 @@ export class MdCheckbox implements ControlValueAccessor {
     this.toggle();
   }
 
+  onFocus(evt: Event) {
+    window.addEventListener('keydown', this._focusWindowKeydownHandler);
```

Comment:
There's actually a special syntax for doing this without directly touching window:
```ts
  host: {
    '(window:domEvent)': 'onWindowEvent($event.type)',
    '(document:domEvent)': 'onDocumentEvent($event.type)',
    '(body:domEvent)': 'onBodyEvent($event.type)'
  }
```